forum/views/comments.py

```diff
@@ -35,10 +35,20 @@ def validate_comments_request_data(data: dict[str, Any]) -> None:
             raise ValueError(f"{field} is missing.")
 
 
+def get_thread_id(parent_comment_id: str) -> str:
+    """
+    The thread Id from the parent comment.
+    """
+    parent_comment = Comment().get(parent_comment_id)
+    if parent_comment:
+        return parent_comment["comment_thread_id"]
+    raise ValueError("Comment doesn't have the thread.")
```

Comment:
no need to raise a value_error as comment_thread_id will always exists, it's a required field.